README.md

```diff
@@ -70,9 +71,7 @@ import wallet from '@shopware-ag/meteor-icon-kit/icons/regular/wallet.svg';
 Take this pseudo html:
 ```html
 <span class="icon-example">
-    <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
-    <!--- ... -->
-    </svg>
+    <wallet />
```

Comment:
Take a look at the full context. The snippet is marked as pseudo anyway and I thought that it's easier to follow since we import wallet.
![grafik](https://user-images.githubusercontent.com/39218577/234218519-42175383-8888-4d2f-8ae3-00477931d615.png)